### DISCUSSION TITLE
This app looks like a superior alternative to "Intellegent Clamp Meter" that I've had to use, but all the DMM's supported are so similar.

### DISCUSSION THREAD

**Discussion by salukikev** [webspiderteam/Bluetooth-DMM-For-Windows#47] [General]
I was specifically looking for a DMM that would support 20A current sensing, and/or other features that are unique.  Unfortunately all the meters I can find in this supported list are pretty conventional in terms of size & specifications.  Are there plans to support any new hardware?  Alternately is there a way to port so that other bluetooth capable meters could utilize this app instead?  The App I mentioned I use with [this clamp meter](https://www.amazon.com/gp/product/B07PY8B1JC) and it is VERY basic.  Essentialy just ports over the display and can save the data stream (RS232).

**Comment by webspiderteam** [comment]
I want to add more models and I already worked on owons t10 model but I didn't shared it because I have no chance to test if it works correct. If your Dmm is using Bluetooth ble protocol i can work on it but I can't do it without testing and collecting enough data. Do you have a chance to help me about this?

**Reply by salukikev** [reply]
Hi there!
Hey, I'm sorry I lost track of this notification email but just remembered it.  I'm happy to volunteer some time to help out of if I can, but my specific agenda has shifted a bit.  Because of the requirements I had (including time) I picked up an[ Owon OW18E meter](https://www.amazon.com/dp/B0B2QVQZR6) which I think is probably a better fit for me in terms of features & specifications but the bluetooth is still not as versatile as it could be.  Also, there are times when I'd like to monitor for more than 20A (per the clamp meter cited above) so now I have both on hand.  I'd love to be able to use one (or both!) with your app instead!

**Reply by webspiderteam** [reply]
I'm busy on these days but I will send you a testing version for owon soon. may be it can work with your new device

**Comment by webspiderteam** [comment]
hi @salukikev,
 I made a test version for owon types. Can you test it? 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/14627920/Portable.zip) portable test  version is here..

**Reply by salukikev** [reply]
Hi!
Happy to report that it seems to work pretty well- I didn't do anything too in-depth, partly because this is a new meter to me and I'm not familiar enough to identify an anomaly vs. normal operation, however at least all assorted function values seem to correctly relay (of course there is no beep for continuity).  The only sligtly odd reading is the intial voltage which appears to be relaying ambient voltage spikes when nothing is connected- when you connect something to it it settles down.   I'll try to use it some more later but for now I'm exicted that it seems to be working pretty well!  Great job!

**Reply by webspiderteam** [reply]
Happy to know that it's working :)  
If devices screen also showing same values and same signs (or similar because I made it for aneng. sings looks and singn can be different but must be same function ofcourse) an if there is no noticeble delay between device display and app display then it should work as it should be. but if there is wrong values or wrong sings and delays inform me please. Because I will wait until comfirmation from you that "Its all working as its suppose to be!". btw I cant find data about true rms sign. if you can take measurements and  send log.txt files to here I can improve stability too. but I forgot to add logging for owon decoder in version that you used. so I added right now. if you use this file logs will be generated.
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/14628542/Portable.zip)

**Comment by webspiderteam** [comment]
Hi @salukikev I made a test version for your  "Digital Clamp Meter AP-570C-APP" that you used with "Intellegent Clamp Meter" app. 

Can you test it? 
test version is here : 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/15284120/Portable.zip)

Please send me log txt files in app folder after test even if its not work correctly. Because I need sample data to test.
